internal/daemon/daemon.go

```diff
@@ -195,7 +200,11 @@ func (d *Daemon) init(listenPort string, schemaExtensions []schema.Update, apiEx
 		resources.InternalEndpoints,
 		resources.PublicEndpoints,
 	}
-	serverEndpoints = append(serverEndpoints, coreEndpoints...)
+
+	for _, server := range d.extensionServers {
```

Comment:
Alright then, perhaps we need a flag to enable/disable unix socket support.

Reply:
I've added a `ServeUnix` flag now. If this is set to true, the resources of the `Server` will be served over the unix socket as well.

If `ServeUnix` is true, then we must also validate that all resource paths are unique across all `Server`s with `ServeUnix=true` or `Address=""`, since they will be aggregated on the unix socket.